src/elements/profiles/Profiles.wc.svelte

```diff
@@ -68,7 +80,7 @@
     // ] },
     { label: "Mnemonics", symbol: "mnemonics", placeholder: "Enter Your Mnemonics", type: "password" },
     // { label: "TFChain Configurations Secret", symbol: "storeSecret", placeholder: "  Secret key used to encrypt your data on TFChain", type: "password" },
-    { label: "Public SSH Key", symbol: "sshKey", placeholder: "Your public SSH key will be added as default to all deployments.", type: "text" },
+    { label: "Public SSH Key", symbol: "sshKey", placeholder: "Disabling the ssh field means you won't be able to access your deployed solutions.", type: "text" },
```

Comment:
can we get the placeholder back to ssh usage, and put the current message as tooltip on the switch?

Reply:
Done.